doc/release-notes/upgrading.en.rst

```diff
@@ -23,14 +23,35 @@ Upgrading to ATS v10.x
 ======================
 
 .. toctree::
-   :maxdepth: 1
 
-Deprecated or Removed Features
-------------------------------
+Feature Changes
+---------------
+
+Removed and Deprecated Features
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 The following features, configurations and plugins are either removed or deprecated in this
 version of ATS. Deprecated features should be avoided, with the expectation that they will be
 removed in the next major release of ATS.
 
+* Removed NPN support
```

Comment:
It's just deprecated at the moment, not removed yet.

Reply:
Updated docs